ios/google_navigation_flutter/Sources/google_navigation_flutter/messages.g.swift

```diff
@@ -906,27 +906,27 @@ struct MarkerAnchorDto: Hashable {
 struct PointOfInterestDto: Hashable {
   /// The Place ID of this POI, as defined in the Places SDK.
   /// This can be used to retrieve additional information about the place.
-  var placeId: String
+  var placeID: String
```

Comment:
Should we use variable naming like this across all ID variables?

Reply:
placeID was named like this before in other places. This will just fix the new POI click endpoint to use same naming.
Changing anything else that already is published to pub.dev will be considered as a breaking change.
But generally, yes, these should be consistent.